README.md

```diff
@@ -68,8 +68,18 @@ GC_THREADS=9
   ```
   
     Note: Don't leave any hyphen/special characters for db_name.
-
-6. There are 2 types of tpcds benchmark script provided,
+	
+4. To verify the TPCDS database creation and row count of tables with expected standard counts for DB. Use the script provided in the utils directory.
+    It can be used if you have chosen DB size - 1GB/1TB/2TB/3TB/10TB/30TB/100TB.
```

Comment:
@nkalband , 1 GB Datasize was earlier causing issues. 
10 GB, 100GB, 200GB datasizes are quite commonly used in our environment. 
Can we get the table counts for those sizes too ?

Reply:
@josiahsams -
I have collected counts for 1TB and 2TB only , rest I updated as per number I got from Kavana.
To get count for  these 3 sizes, I will have to create  these tpcds DB first else If you can provide numbers, I can add them.

Reply:
@nkalband , Since these are small size DB's, kindly create those DB and update the table row count.

Reply:
Updated record data for 10G, 100G and 200G